src/lib/parts.ts

```diff
@@ -249,11 +322,22 @@ export class NodePart implements Part {
         node.nodeType === 3 /* Node.TEXT_NODE */) {
       // If we only have a single text node between the markers, we can just
       // set its value, rather than replacing it.
-      // TODO(justinfagnani): Can we just check if this.value is primitive?
-      (node as Text).data = value as string;
+      if (sanitizeDOMValue) {
+        value = String(sanitizeDOMValue(value, 'data', 'property', node));
```

Comment:
why `'data'` here and `'textContent'` below?

Reply:
Switched to data in both cases